src/helpers/database.ts

```diff
@@ -0,0 +1,14 @@
+import { connect } from "mongoose";
+import { Config } from "../config";
+import { logger } from "../logger";
+
+export const connectDatabase = async () => {
+    //let database_url = Config?.database_url
+    let database_url = "mongodb+srv://discordTest:discordTestPassword@thangtran.ckqo8.mongodb.net/discord_test?retryWrites=true&w=majority"
```

Comment:
It's just for testing, I'll remove the user after it's done, so no harm

Reply:
Ok. One thing to mention is that there are requirements for which DB is used. Postures and SQLite, depending on environment